src/main/java/uk/gov/hmcts/reform/sscs/exceptionhandlers/ResponseExceptionHandler.java

```diff
@@ -16,19 +17,26 @@
     private static final Logger log = LoggerFactory.getLogger(ResponseExceptionHandler.class);
 
     @ExceptionHandler(UnauthorizedException.class)
-    protected ResponseEntity<Void> handleUnAuthorizedException(Exception exception) {
+    protected ResponseEntity<Void> handleUnAuthorizedException(UnauthorizedException exception) {
         log.error(exception.getMessage(), exception);
 
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
     }
 
     @ExceptionHandler(ForbiddenException.class)
-    protected ResponseEntity<Void> handleForbiddenException(Exception exception) {
+    protected ResponseEntity<Void> handleForbiddenException(ForbiddenException exception) {
         log.error(exception.getMessage(), exception);
 
         return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
     }
 
+    @ExceptionHandler(FeignException.class)
+    protected ResponseEntity<Void> handleFeignException(FeignException exception) {
+        log.error(exception.getMessage(), exception);
+
+        return ResponseEntity.status(exception.status()).build();
```

Comment:
any reason you aren't adding the exception message here?

Reply:
added now